front/app/components/HookForm/ColorPicker/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {
+  ColorPickerInput,
+  ColorPickerInputProps,
+} from '@citizenlab/cl2-component-library';
+import Error, { TFieldName } from 'components/UI/Error';
+import { Controller, useFormContext } from 'react-hook-form';
+import { CLError } from 'typings';
+
+interface Props
+  extends Omit<ColorPickerInputProps, 'value' | 'onChange' | 'type'> {
+  name: string;
+}
+
+const ColorPicker = ({ name, ...rest }: Props) => {
+  const {
+    formState: { errors },
+    control,
+  } = useFormContext();
+
+  const defaultValue = '';
+
+  const validationError = errors[name]?.message as string | undefined;
+
```

Comment:
It's kind of a shame that we have to do all this type casting here... I guess you already looked into making this work better with TS?